metrics.py

```diff
@@ -55,21 +93,31 @@ def __init__(self, elapsed_millis, start_counters, end_counters):
       start_counters.get(SECTORS_WRITTEN_KEY, 0.))
     self.millis_writing = (end_counters.get(MILLIS_WRITING_KEY, 0.) -
       start_counters.get(MILLIS_WRITING_KEY, 0.))
+    self.total_io_millis = (end_counters.get(TOTAL_IO_MILLIS_KEY, 0.) -
+      start_counters.get(TOTAL_IO_MILLIS_KEY, 0.))
+    self.__calculate_util_and_throughput()
 
   def add_metrics(self, other_metrics):
     self.elapsed_millis += other_metrics.elapsed_millis
     self.bytes_read += other_metrics.bytes_read
     self.millis_reading += other_metrics.millis_reading
```

Comment:
Can you add a note by this and millis_writing to point out that they may not be accurate?